saylua/modules/search/templates/results.html

```diff
@@ -1,7 +1,7 @@
 {% extends "layout.html" %}
 {% set page_title = "Search Results" %}
 {% block body %}
-<h2>Search Saylua{% if query %}: {{query}}{% endif %}</h2>
+<h1>Search Saylua{% if query %}: {{query}}{% endif %}</h1>
```

Comment:
Oh, well there's `{{ ": {}".format(query) if query else "" }}` I suppose. That's not really very far from the original though.